e2e/journeys/citizen/createCase/C100/subJourneys/C100People1.ts

```diff
@@ -1,26 +1,29 @@
 import { Page } from "@playwright/test";
-import { UserRole } from "../../../../../common/types";
 import { AddChildrenPage } from "../../../../../pages/citizen/createCase/C100/people/addChildrenPage";
 import { ProvideDetailsPage } from "../../../../../pages/citizen/createCase/C100/people/provideDetailsPage";
 import { WhichDecisionsPage } from "../../../../../pages/citizen/createCase/C100/people/whichDecisionsPage";
+import { ParentalResponsibilityPage } from "../../../../../pages/citizen/createCase/C100/people/parentalResponsibilityPage";
+import { FurtherInformationPage } from "../../../../../pages/citizen/createCase/C100/people/furtherInformationPage";
+import { HasOtherChildrenPage } from "../../../../../pages/citizen/createCase/C100/people/hasOtherChildrenPage";
 import { ApplicantGender } from "../../../../../common/types";
+import { yesNoDontKnow } from "../../../../../common/types";
 
 interface C100People1Options {
   page: Page;
-  user: UserRole;
   accessibilityTest: boolean;
   errorMessaging: boolean;
   gender: ApplicantGender;
+  c100PeopleYesNoDontKnow: yesNoDontKnow;
   subJourney: boolean;
 }
 
 export class C100People1 {
   public static async c100People1({
     page,
-    user,
     accessibilityTest,
     errorMessaging,
     gender,
+    c100PeopleYesNoDontKnow,
     subJourney,
   }: C100People1Options): Promise<void> {
     if (subJourney) {
```

Comment:
If there's no need for a subjourney then I reckon we just remove the arg + block